src/main/java/com/bokov/FirstTGBot/models/BTC.java

```diff
@@ -0,0 +1,56 @@
+package com.bokov.FirstTGBot.models;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+
+@Entity
+@Table(name = "BTC")
+public class BTC {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "prise")
+    private double prise;
+
+    @Column( name = "date_of_request")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date dateOfRequest;
+
+    public BTC() {
+    }
+
+    public BTC(double prise, Date dateOfRequest) {
+        this.prise = prise;
+        this.dateOfRequest = dateOfRequest;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public double getPrise() {
+        return prise;
+    }
+
+    public void setPrise(double prise) {
+        this.prise = prise;
+    }
+
+    public Date getDateOfRequest() {
+        return dateOfRequest;
+    }
+
+    public void setDateOfRequest(Date dateOfRequest) {
```

Comment:
I would suggest using lombok annotations instead of manual generation of getter/setter